visualization/dashboard/src/MLIDashboard/Controls/CohortEditor/CohortEditor.styles.ts

```diff
@@ -246,7 +243,7 @@ const cohortEditor = cohortEditorStyles();
 export const cohortEditorCallout: () => ICalloutContentStyles = () => {
     return {
         container: {},
-        root: {},
+        root: {marginLeft:"212px"},
```

Comment:
This seems better in positioning, and we can definitely check it in for the demo, but we should check with Minsoo if she wants it to always be horisontally centered, and if so, we can go over how to do that, later